doc/nrf/known_issues.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
```

Comment:
You also need to put the page back in the ToC (in index.rst)

Reply:
And also the changelog.

Reply:
Ah, of course!